File: 2018/3회/1번.java
//다음은 피보나치 수열의 합계를 구하는 프로그램을 Java 언어로 구현한 것이다. 프로그램을 분석하여 그 실행 결과를 쓰시오.
public class Main {
	public static void main(String[] args) {
    int a, b, c, sum;
    a = b = 1;
    sum = a + b;
    for(int i = 3; i <=5; i++) {
      c = a + b;
      sum += c;
      a = b;
      b = c;
    }
    System.out.println(sum);
  }
}

//답
//12

File: 예상문제/1번.c
//다음은 C언어 코드이다. 출력 결과를 쓰시오.
#include <stdio.h>
void main() {
  int i, j;
  int a[3][2] = { {1,2}, {3,4}, {5,6} };
  int sum = 0;
  for (i = 0; i < 3; i++) {
    for (j = 0; j < 2; j++) {
      sum += a[i][j];
    }
  }
  printf("%.2f", (float)sum/(3*2));
}

//답
//3.50
